watson/watson.py

```diff
@@ -302,9 +305,16 @@ def stop(self, stop_at=None):
         if stop_at > arrow.now():
             raise WatsonError('Task cannot end in the future.')
 
+        if note is None:
+            note = old.get('note')
+        elif old.get('note') is not None:
+            print('Overwriting old note:\n>> {}'.format(old.get('note')))
```

Comment:
This means that this warning is printed first, before the always printed watson stop message. This is a bit visually jarring, but it might be a good thing since it is an important thing to note, see screenshot. I tried passing a variable up to watson cli and then use echo after the stop message, but could not find a good way of doing this since the frameattributesare actually properties are note modifieable so I could not pass say a tuple and then print the old note and overwrite the property with the new note.

![image](https://user-images.githubusercontent.com/4560057/85078526-73ca0500-b179-11ea-8a5a-87866cc83c4f.png)